tests/system/README.md

```diff
@@ -0,0 +1,72 @@
+# Layer0 System Tests
+
+### Test Architecture
+Each test case contains two components: a subdirectory under `cases/`, and a `<case name>_test.go` file.
+The test case directory contains terraform files that outline the resources needed for the test. 
+The testing framework will create these resources with `terraform apply` before executing the test, and delete them with `terraform destroy` once the test has completed.
+
+Note that the `cases/modules` directory is not a test case directory.
+It holds our [terraform modules](https://www.terraform.io/docs/modules/usage.html) to reduce duplicate code in our terraform files.
+
+If your test case was named `my_simple_example`, you should create the following files and directories:
+```
+layer0/tests/system
+|-- mySimpleExample_test.go
+|-- cases
+|   |-- my_simple_example
+|   |    |-- main.tf
+```
+
+### Writing a Test
+Writing a test is relatively straightforward.
+Following the example above, the contents of `mySimpleExample_test.go` should look like:
+```go
+package system
+
+import (
+        "testing"
+)
+
+func TestMySimpleExample(t *testing.T) {
+    c := startSystemTest(t, "cases/my_simple_example", nil)
+    defer c.Destroy()
+    
+    // the rest of your test
+}
+```
+
+The `startSystemTest` function tells the test to run in parallel, creates a new `framework.SystemTestContext` object using the specified directory (`"cases/my_simple_example"` in this example), and runs `terraform apply`.
+
+In addition to wrapping terraform commands, the `SystemTestContext` object contains some wrappers around the `cli/client` package, e.g.:
+```
+env := c.GetEnvironment("environment_name")
+svc := c.GetService("environment_name", "service_name")
+lb := c.GetLoadBalancer("environment_name", "load_balancer_name")
+```
+
+
+### Running a Test
```

Comment:
This whole doc is great :1st_place_medal: 

My only comment for here would be to define what we consider a 'system test' to actually be. This is probably best illustrated by whatever we have in `/cases` but there is still ambiguity around what a system test versus end-to-end test versus integration test. From what i'm seeing, we fall somewhere in the realm of system test because we're testing module functionality and not necessarily _workflow_.